osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/datadecoder/dlmsclassdatadecoder/TimeAndEventsClassesDecoder.java

```diff
@@ -34,6 +38,33 @@ public TimeAndEventsClassesDecoder(
     this.basicDlmsDataDecoder = basicDlmsDataDecoder;
   }
 
+  // Class-id 8: Clock
```

Comment:
To be complete: // Class-id 8: Clock Attribute-id 4

Reply:
Completed. Also for attributes of other classes. And sorted according to attribute id for each class.